### PR TITLE
Add ZeroTier, Tailscale and Netbird

### PR DESCRIPTION
**Pull request by Alerymin**
Zerotier: Easy
Tailscale: Limited (it's very hard to understand, the page linked is the best one I could find)
NetBird: Easy

### CODE REVIEW

**Comment by tupaschoal** [comment]
There are a few failing checks, can you tend to them please? @Alerymin

**Comment by Alerymin** [comment]
Sorry I completely forgot. I fixed it

File: _data/sites.json [hunk 1]
```diff
@@ -8867,7 +8867,7 @@
         "difficulty": "hard",
         "notes": "It is also possible to file an account deletion request via the \"Contact Us\" section at the bottom of the page.",
         "domains": [
-            "glovoapp.com"
+            "glovoapp.https://tailscale.com/kb/1237/delete-tailnetcom"
```
Comment (tupaschoal) [inline _data/sites.json:8870]: This seems like badly copy+pasted stuff?
```suggestion
            "glovoapp.com"
```

**Review by tupaschoal — COMMENTED** [review]
Seems fine, just one minor line note

File: _data/sites.json (hunk 1 above)
Comment (Alerymin) [reply _data/sites.json:8870]: Completely, my bad

**Comment by Alerymin** [comment]
I changed it myself because for some reason github refused to commit your fix

**Comment by tupaschoal** [comment]
> I changed it myself because for some reason github refused to commit your fix

No problem, this has been happening (and I can't understand why)

**Comment by tupaschoal** [comment]
Thanks for your contribution!

**Pull request merged**